src/libraries/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs

```diff
@@ -145,6 +145,7 @@ public async Task TestBasicReceiveAndResponseEvents()
             }
         }
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/112792")]
```

Comment:
I almost missed that this test is also failing. It seems to be a [product bug](https://github.com/dotnet/runtime/issues/112792), disabling for now. cc @tarekgh in case you have concerns.